src/ol/dom/dom.js

```diff
@@ -298,3 +298,51 @@ ol.dom.transformElement2D =
     // content size.
   }
 };
+
+
+/**
+ * Get the current coordinates of the given element relative to the document.
+ * Equivalent to jQuery's `$(el).offset()`.
+ * @param {!Element} element Element.
+ * @return {ol.Pixel}
+ */
+ol.dom.offset = function(element) {
+  var rect = element.getBoundingClientRect();
+
+  return [
+    rect.left + goog.global.document.body.scrollLeft,
+    rect.top + goog.global.document.body.scrollTop
+  ];
+};
```

Comment:
I used the implementation from [You Might Not Need jQuery](http://youmightnotneedjquery.com/#offset), which also supports IE 8.

But `goog.style.getPageOffset` works as well, so we don't need our own function, like you said.